BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsDatabase.kt

```diff
@@ -8,11 +8,14 @@ import androidx.room.RoomDatabase
 
 // Ref: https://developer.android.com/training/data-storage/room/migrating-db-versions
 @Database(
-        version = 3,
+        version = 4,
         entities = [AnalyticsEvent::class],
         autoMigrations = [
             AutoMigration(from = 1, to = 3),
-            AutoMigration(from = 2, to = 3)
+            AutoMigration(from = 2, to = 3),
+            AutoMigration(from = 1, to = 4),
```

Comment:
Since we already have migration from 1 to 2 and 2 to 3, please only add 3 to 4. Otherwise, the increase in auto-migrations we have to provide would increase non-linearly.

Reply:
I'm thinking we might want to consider using SQL to store the analytics JSON as a string instead of typed values to make adding analytics tags easier in the future. Do y'all think it makes sense to do this now?

Reply:
It does seem like [more events are coming](https://github.com/braintree/braintree_ios/pull/1292) to track latency.

Reply:
I think that would be a separate ticket?

Reply:
Yes it would be a separate task. I created DTMOBILES-619 for exploring different storage options.

Reply:
Thank you Jax!